### DISCUSSION TITLE
hashrate is keep falling on esp32 dual core

### DISCUSSION THREAD

**Discussion by ahmedhussien42** [revoxhere/duino-coin#349] [Show and tell]
hi there
hashrate is keep falling down on esp32 dual core code till disconnecting from the server,  without any known reason to me - tried to mine with 5 different chips and the result is the same 

any help !!!!
thanks in advance

**Comment by revoxhere** [comment]
As from JoyBed said before, something's wrong with the Wifi library. We are working on it

**Reply by ahmedhussien42** [reply]
thanks for your fast reply

**Reply by Moayidd** [reply]
I have the exact same issue with the two ESP32's on which I'm running release 2.4. I had the same issue with release 2.3.  Waiting for your fix

**Comment by Tech1k** [comment]
Closing this discussion in a few as the new release should've fixed this and there hasn't been activity in over a month.
